docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,101 +1,119 @@
+      }
+    }
+```
 
-Site administrators have selective control over users' access to Cody Enterprise, which is managed via the Sourcegraph role-based access control system. This provides a more intuitive user interface for assigning permission to use Cody.
+- Cody is enabled on your self-hosted Sourcegraph enterprise instance
 
-### Analytics
+## Disable Cody
 
-<Callout type= "note">Cody Analytics are supported with VS Code IDE extension and on the latest versions of JetBrains IDEs.</Callout>
+To turn Cody off:
 
-Cody Enterprise users can view analytics for their instance. A separately managed cloud service for Cody analytics handles user auth, gets metrics data from Sourcegraph's BigQuery instance, and visualizes the metrics data.
+- Go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
 
-The following metrics are available for Cody Enterprise users:
+```json
+    {
+      // [...]
+      "cody.enabled": false
+    }
+```
 
-| **Metric Type** |                                                                                                                                           **What is measured?**                                                                                                                                            |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Active users    | - Total active users <br /> - Average daily users <br /> - Average no. of days each user used Cody (of last 30 days) <br /> - Cody users by day (last 30 days) <br /> - Cody users by month (last two months) <br /> - Cody users by number of days used                                                   |
-| Completions     | - Total accepted completions <br /> - Minutes saved per completion <br /> - Hours saved by completions <br /> - Cody completions by day <br /> - Completions acceptance rate <br /> - Weighted completions acceptance rate <br /> - Average completion latency <br /> - Acceptance rate by language <br /> |
-| Chat            | - Total chat events <br /> - Minutes saved per chat <br /> - Hours saved by chats <br /> - Cody chats by day                                                                                                                                                                                               |
-| Commands        | - Total command events <br /> - Minutes saved per command <br /> - Hours saved by commands <br /> - Cody commands by day <br /> - Most used commands                                                                                                                                                       |
+- Next, remove `completions` configuration if they exist
```

Comment:
Nit: The previous step kinda implies this by not listing the "completions" section. Maybe it'd be more elegant to not list this as a bullet but just reiterate that yes, removing that config is required.

Is it, though? Like, is it not a valid state if they leave the `completions` config in place? As a customer, I'd appreciate if this was clarified either way, so:
a.) by saying "Note that you _must_ remove the `completions` section" to after disabling Cody to have a valid config."
or
b.) by saying "You can leave the `completions` section in place if you, for example, may want to enable Cody again later."

If the latter is possible, I think it's nicer.

Reply:
cc: @MaedahBatool 

For what it’s worth, from an implementation perspective, if both `completions` and `modelConfiguration` fields are specified, the `modelConfiguration` field takes precedence.

Reply:
Yeah, but this section is about disabling Cody. Once Cody is disabled, the precedence between those keys doesn't matter here. So the question is really whether the config is still valid if this key exists but Cody is disabled.

Reply:
You’re absolutely right, David — apologies for the confusion! 
@MaedahBatool, in this case, we can either mention removing both fields (`completions` and `modelConfiguration`) or simply skip this part, as setting `"cody.enabled": false` alone is sufficient to disable Cody on the instance.

Reply:
Done done 👍